src/Components/IndividualArticle.jsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getIndividualArticle } from "../api";
+import formatDate from "../utils/utils";
+import ArticleInfoBar from "./ArticleInfoBar";
+
+const IndividualArticle = () => {
+  const {article_id} = useParams();
+  const [article, setArticle] = useState("");
```

Comment:
It is best practice to have the state set to an empty version of the data type we expect it to be once it is set